brion/plugin/compartmentReportHDF5.cpp

```diff
@@ -143,7 +187,10 @@ bool CompartmentReportHDF5::handles(const CompartmentReportInitData& initData)
 std::string CompartmentReportHDF5::getDescription()
 {
     return "SONATA HDF5 compartment reports:"
-           "  [file://]/path/to/report.(h5|hdf5)";
+           "  "
+           "[file://]/path/to/"
+           "report.(h5|hdf5)[?[cache_size=(auto|num_bytes):][cell_to_frames="
```

Comment:
code uses cell**S**_to_frames

Reply:
maybe try to put spaces differently to fix this horrendous clang formatting here?

Reply:
It got formatted that way because I had a single string. If I split it by hand, clang-format won't break it.